projects/1.20/assets/goblins-tyranny/goblins_tyranny/lang/zh_cn.json

```
@@ -0,0 +1,887 @@
+{
+    "advancements.bag_success.descr": "打开一个炸弹包",
+    "advancements.bag_success.title": "神秘的包",
+    "advancements.barrel_success.descr": "在木桶里找到一只哥布林",
+    "advancements.barrel_success.title": "有点可疑",
+    "advancements.blazing_liquor_success.descr": "喝一次灼燃烈酒",
+    "advancements.blazing_liquor_success.title": "这很棒！",
+    "advancements.bomb_success.descr": "获得一颗炸弹",
+    "advancements.bomb_success.title": "哔，邦，嘣！",
+    "advancements.deadly_liquor_success.descr": "喝一次致命烈酒",
+    "advancements.deadly_liquor_success.title": "我觉得不好......",
+    "advancements.droblin_success.descr": "获得一个多布林",
+    "advancements.droblin_success.title": "永远实用的装置",
+    "advancements.engineer_success.descr": "使用一次工程师工作台",
+    "advancements.engineer_success.title": "这是戈布林·马斯克",
+    "advancements.fire_success.descr": "获得一颗燃烧炸弹",
+    "advancements.fire_success.title": "现在是棉花糖时间",
+    "advancements.glitteron_armor_success.descr": "为你的闪光崽获得一件盔甲",

```

```suggestion
    "advancements.glitteron_armor_success.descr": "获得一件为闪光崽打造的盔甲",
```
an armor for your Glitteron 这里其实用“的”就行